sdk/couchbase-core/src/memdx/error.rs

```diff
@@ -451,10 +478,61 @@ pub enum ServerErrorKind {
     UnknownScopeName,
     #[error("collection name unknown")]
     UnknownCollectionName,
+    #[error("subdoc error")]
```

Comment:
It probably makes sense to make this transparent and just have the message from the underlying error kind?

Reply:
Done